fbw-a32nx/src/systems/fmgc/src/navigation/Navigation.ts

```diff
@@ -53,4 +53,7 @@ export class Navigation {
 
         this.groundSpeed = SimVar.GetSimVarValue('GPS GROUND SPEED', 'knots');
     }
+
+    ///  FEATURE TO BE IMPLEMENTED:
+    ///  NOTE : When FMC internal position will be coded, there needs to be a way to make the FMC lose its position in flight when switching database in the A320_Neo_CDU_IdentPage.js
```

Comment:
To be confirmed really... with GPIRS available this shouldn't be the case as it does not depend on the nav database...

Reply:
If you have a clear view of what needs to be done to reproduce the mentions of "loss of position and ND blank" in the FMC manual, please do advise.
I was myself wondering if it had to do with the alignment of the three ADIRS that is done in the FMC and that would lead to loss of FMC position and ND blank but also was told no.

Reply:
Definitely won't lose IR alignment. The underlying FM navigation function is under development so not really ready for this yet.

Reply:
Should we close this conversation or do you need me to do something specific ?

Reply:
Nothing to be done at this stage 👍.